tests/test_actinia_mapset_management.py

```diff
@@ -32,19 +32,32 @@
 
 from actinia import Actinia
 from actinia.mapset import Mapset
+from actinia.region import Region
 
 from .mock.actinia_mock import (
     ACTINIA_BASEURL,
     version_resp_text,
     location_get_mapset_resp,
 )
+
 from .mock.actinia_location_management_mock import get_locations_resp
 
+from .mock.actinia_mapset_management_mock import (
+    mapset_get_info_resp,
+    mapset_creation_resp,
+    delete_mapset_resp
+)
```

Comment:
```suggestion
from .mock.actinia_mapset_management_mock import (
    get_locations_resp,
    mapset_get_info_resp,
    mapset_creation_resp,
    delete_mapset_resp
)
```

Reply:
I'm importing `get_locations_resp` from `.mock.actinia_location_management_mock` because I didn't want to duplicate the response code in `.mock.actinia_mapset_management_mock`, but I'll add `get_locations_resp` to `.mock.actinia_mapset_management_mock` if that is the pattern you want to follow.